app/routes/members-area.my-orders/route.module.scss

```diff
@@ -1,4 +1,66 @@
-.underConstructionMessage {
-    text-align: center;
+@import '@styles/variables.scss';
+
+.title {
     font: var(--heading5);
+    font-size: 26px;
+}
+
+.message {
+    margin-top: 13px;
+}
+
+.orders {
+    margin-top: 35px;
+}
+
+.orderListHeader,
+.orderHeader {
+    height: 78px;
+    display: flex;
+    align-items: center;
+    justify-content: stretch;
+    gap: 20px;
+}
+
+.orderListHeader > div,
```

Comment:
these styles looks strange, can we add some class name? what is that? maybe columnTitle?

Reply:
I didn't want to pollute jsx too much, as there 4 divs inside of a row.. but ok :)